website/docs/language/syntax/configuration.mdx

```diff
@@ -73,6 +73,15 @@ resource "aws_instance" "example" {
 }
 ```
 
+```hcl
```

Comment:
```suggestion
Use the following syntax to configure Terraform blocks:  
```hcl
```
We should add a sentence to introduce the syntax. It would actually be better to move this section above the example and add a sentence after it to set up the example. Does that make sense?